docs/comparison.md

```diff
@@ -81,12 +81,13 @@ hoisting all necessary data requirements to a page-wide query.
 |                               | urql                             | Apollo             | Relay             |
 | ----------------------------- | -------------------------------- | ------------------ | ----------------- |
 | React Bindings                | ✅                               | ✅                 | ✅                |
-| React Hooks Support           | ✅                               | ✅                 | ✅ (experimental) |
+| React Hooks Support           | ✅                               | ✅                 | ✅ (experimental), 🟡 `relay-hooks` |
```

Comment:
I'd say to let the first-class integration like it is

Reply:
Exactly but it is marked as experimental. While relay-hooks is not experimental and supports both react concurrent and legacy modes

Reply:
We sort and filter these comparison entries in order of priority, so if there's a first party solution, nothing else will be added,
If there's a custom entry in the docs no third-party solution will be added, if there's a third party solution that's commonly used or in third party docs, that's added, and so on.

That's because we can't add every possible note here regarding _how_ somethings achieved

Reply:
Two entries can be created:

|                                            | urql                                | Apollo                                        | Relay                          |
| ------------------------------------------ | ----------------------------------- | --------------------------------------------- | ------------------------------ |
| React concurrent Hooks Support     | 🛑                               | 🛑                 | ✅ (experimental) |
| React Legacy Hooks Support           | ✅                               | ✅                 |  🟡 `relay-hooks` |

Reply:
That's a good option 👍 let's do that. `urql` does support concurrent mode but I'm not 100% sure about Apollo. Let's just assume for now that they do